doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -71,6 +71,7 @@ Added
 * Clock control is enabled in SDFW.
 * Global domain power request service is integrated in SDFW.
 * PUF values from SDROM are cleared on boot.
+* The SUIT cache can now be defined in Nordic manifests.
```

Comment:
This should be the only entry compared to 10.1.0, isn't that listed here?

Reply:
@hakonfam would you like mentions of 10.1.0 and 10.2.0 separated? Currently they are just listed as 10.2.0

Reply:
I guess the best would be to separate them, as that could help describe issues in older versions.